src/components/DataTable/DataTableContext/DataTableContextProvider.tsx

```diff
@@ -58,7 +62,7 @@ export const DataTableContextProvider: FC<PropsWithChildren<DataTableProps>> = m
 
         const tableWidth = useMemo(() => {
             return columns.reduce(
-                (acc, column) => acc + (columnWidths?.[column] ?? defaultColumnWidth),
+                (acc: number, column) => acc + (columnWidths?.[column] ?? defaultColumnWidth),
```

Comment:
Later, we can create a hook to extend this so that in groupby we can have extra width based on number of nested groups